doc/getting-started.rst

```diff
@@ -168,53 +168,20 @@ Windows 11
 
 Windows 11 is supported, but has not been tested to the same level as Windows 10.
 
-Vulkan SDK
-----------
-
-For Vulkan support the Vulkan SDK is required to be installed. It can be
-acquired from `LunarG <https://www.lunarg.com/vulkan-sdk/>`_. The latest version
-should always be supported. The minimum supported version is 1.1.97.
-
-The Vulkan SDK includes `SPIRV-Tools`_ as part of the package. Therefore
-`SPIRV-Tools`_ does not need to be installed separately.
-
-Package Manager Installation
-............................
 
-Users of Ubuntu 20.04 and later should configure their apt package manager to
-install the Vulkan SDK packages directly as described by the `LunarG Vulkan SDK
-packages`_ page. This installation method does neither set nor require the
-:envvar:`VULKAN_SDK` environment variable, avoiding the configuration
-complexities experienced when installing manually.
 
-Windows users should install directly from `LunarG
-<https://www.lunarg.com/vulkan-sdk/>`_, as at the time of writing the Vulkan SDK
-cannot be installed through Chocolatey.
-
-Manual Installation
-...................
-
-Other Linux users will need to take some extra steps after downloading the
-tarball distribution. First run the ``build_tools.sh`` script; then source the
-``setup-env.sh`` script whilst the Vulkan SDK directory as the current
-directory. These steps are needed to access the ``spirv-as`` tool through the
-:envvar:`VULKAN_SDK` environment variable in order to successfully compile
-the oneAPI Construction Kit and generate all test suites correctly.
-
-.. hint::
-   Modifying ``.bashrc`` to source the ``setup-env.sh`` script avoids having to
-   remember to do this on every new terminal session building the oneAPI
-   Construction Kit.
-
-The ``setup-env.sh`` script prepends :envvar:`VULKAN_SDK` to the system
-:envvar:`PATH`, leading to prioritization of :envvar:`VULKAN_SDK` binaries
-ahead of any other versions of the tools which may be installed in ``/usr/bin``.
 
 SPIRV-Tools
 -----------
 
-When OpenCL support is required but Vulkan support is not, `SPIRV-Tools`_
-**may** optionally be installed on the system :envvar:`PATH`. This is
+SPIRV-Tools is needed to build the oneAPI Construction Kit with the SPIR-V
+testing, otherwise those tests will be disabled. This can either come from
+Vulkan SDK OR installing SPIRV-Tools directly. Windows users should install
+directly from `LunarG <https://www.lunarg.com/vulkan-sdk/>`_, as at the time of
+writing the Vulkan SDK cannot be installed through Chocolatey. On Ubuntu,
+installing spirv-tools directly is recommended.
```

Comment:
I'm not sure this paragraph is needed, I think just the next paragraph is enough since users get a fully featured OCK build regardless of whether SPIRV-Tools are available. If we do want to keep this, it doesn't read right, it recommends too strongly to install this, it still makes it seem as if it's required.